assignments/callbacks.js

```diff
@@ -21,32 +21,63 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
+  
+  return cb(arr.length)
+
   // getLength passes the length of the array into the callback.
 }
 
 function last(arr, cb) {
+    // items[0] ; how to get the first item
+     return cb(arr[arr.length -1])
+
   // last passes the last item of the array into the callback.
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+//   let a = 3
+// function addTwo(x) {
+//   let ret = x + 2
+//   return ret
+//  }
+//  let b = addTwo(a)
+//  console.log(b);
+
+  return cb(arr[x + y]);
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(arr[x * y]);
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  const  contains = list.length;
```

Comment:
this won't work for many reason including name collision and missing logic. you need to do something like.

```javascript
return cb(list.includes(item));
```